Apps/Src/SendTritium.c

```diff
@@ -0,0 +1,585 @@
+/**
+ * @copyright Copyright (c) 2022 UT Longhorn Racing Solar
+ * 
+ * @file SendTritium.c
+ * @brief Function implementations for the SendTritium application.
+ * 
+ * This contains functions relevant to updating the velocity and current setpoitns
+ * of the Tritium motor controller. The implementation includes a normal current
+ * controlled mode, a one-pedal driving mode (with regenerative braking), and cruise control.
+ * The logic is determined through a finite state machine implementation.
+ * 
+ * If the macro __TEST_SENDTRITIUM is defined prior to including SendTritium.h, relevant
+ * variables will be exposed as externs for unit testing.
+ * 
+ * @author Nathaniel Delgado (NathanielDelgado)
+ * @author Diya Rajon (diyarajon)
+ * @author Ishan Deshpande (IshDeshpa, ishdeshpa@utexas.edu)
+ */
+
+#include "Pedals.h"
+#include "ReadCarCAN.h"
+#include "Minions.h"
+#include "SendTritium.h"
+#include "ReadTritium.h"
+#include "CANbus.h"
+#include "UpdateDisplay.h"
+
+// Macros
+#define MAX_VELOCITY 20000.0f // rpm (unobtainable value)
+#define MIN_CRUISE_VELOCITY 20.0f // m/s
+#define MAX_GEARSWITCH_VELOCITY 8.0f // m/s
+
+#define BRAKE_PEDAL_THRESHOLD 5  // percent
+#define ACCEL_PEDAL_THRESHOLD 10 // percent
+
+#define ONEPEDAL_BRAKE_THRESHOLD 25 // percent
+#define ONEPEDAL_NEUTRAL_THRESHOLD 35 // percent
+
+#define GEAR_FAULT_THRESHOLD 3 // number of times gear fault can occur before it is considered a fault
+
+#ifdef __TEST_SENDTRITIUM
+#define STATIC(def) static def
+#else
+#define STATIC(def) def
+#endif
+
+// Inputs
+STATIC(bool cruiseEnable = false;)
+STATIC(bool cruiseSet = false;)
+STATIC(bool onePedalEnable = false;)
+STATIC(bool chargeEnable = false;)
```

Comment:
```suggestion
STATIC(bool) cruiseEnable = false;
STATIC(bool) cruiseSet = false;
STATIC(bool) onePedalEnable = false;
STATIC(bool) chargeEnable = false;
```
why not something like this? having the whole line inside the macro func is ugly